administrator/components/com_banners/helpers/html/banner.php

```diff
@@ -111,6 +111,6 @@ public static function pinned($value, $i, $enabled = true, $checkbox = 'cb')
 			),
 		);
 
-		return JHtml::_('jgrid.state', $states, $value, $i, 'banners.', $enabled, true, $checkbox);
+		return JHtml::_('jGrid.state', $states, $value, $i, 'banners.', $enabled, true, $checkbox);
```

Comment:
Short term it'd be easier to just fix the class declaration in `JHtmlJGrid` to be `JHtmlJgrid` (as that matches the expected class name based on JLoader).